packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -144,7 +148,6 @@ export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
   'ca-central-1': '840364872350',
   'eu-central-1': '840364872350',
   'eu-north-1': '840364872350',
-  'eu-south-1': '840364872350',
```

Comment:
I did merge `master` into this PR. Could not find references to `eu-south-1` in the list of supported regions... but I could have missed it?

Reply:
It's here: https://github.com/aws/aws-cdk/blob/fe8d5e67f9e53c8f74a304e27cb668354699cc48/packages/%40aws-cdk/region-info/build-tools/fact-tables.ts#L147-L147

Any reason it's removed in this PR @dfezzie ?